### DISCUSSION TITLE
Cant proof my school id

### DISCUSSION THREAD

**Discussion by privt00** [community/community#126498] [GitHub Education]
Hey, i want to acces githubs student pack but have no studentd id i just have a email adress of the scool but github asks me to also upload a id i uploaded a class shelude but also not working can someone pls help me. Thx for every response.

**Comment by namdng09** [comment]
Hi there! I have got approved by github to submit request.

![image](https://github.com/community/community/assets/144111926/45ce022f-b670-4247-9a88-9a2056b28d6b)

I think there many way to vertify academic status.

![image](https://github.com/community/community/assets/144111926/6063873d-5618-4765-a9df-cfe07addd0b2)

You said that you did't have student id, you can upload Dated syllabus for a class...

BUT i think, you should go to school and ask for a official transcript. The transcript cost may be free or very small price. The success rate will be higher.

Official transcript will look like this:
![image](https://github.com/community/community/assets/144111926/bb3af928-9bd7-4103-bdf1-810ca7a32f15)

NOTE: Vertify your academic email address on your GitHub account, if your school provides one.
Secure your GitHub account with two-factor authentication.

I hope you like this comment.
Thankyou for reading and have a good day.

**Reply by privt00** [reply]
Hey, thank you for your Response but on my screen i dont see the option to select the type of proof.
